app/src/lib/app-state.ts

```diff
@@ -418,6 +418,16 @@ export interface IRepositoryState {
   readonly compareState: ICompareState
   readonly selectedSection: RepositorySectionTab
 
+  /**
+   * The state of the current pull request view in the repository.
+   *
+   * It will populated when a user initiates a pull request. It may have
+   * content to retain a users pull request state if they navigate
+   * away from the current pull request view and then back. It is returned
+   * to null after a pull request has been opened.
```

Comment:
…or the dialog is closed, right?

Reply:
Maybe... what if the user starts a PR and fills out the body but then gets a notification about another PR.. maybe we want to remember where they are at and let them get back to it?

Reply:
Yeah in that case, it'd be nice to keep it.